test/integration/issues/portal/FACES-1427-portlet/src/test/java/com/liferay/faces/test/FACES1427PortletTest.java

```diff
@@ -293,7 +293,7 @@ public void steps89012() throws Exception {
 
 		// type the word 'subsequent'
 		textarea1.sendKeys("subsequent");
-		Thread.sleep(50);
+		Thread.sleep(100);
```

Comment:
@stiemannkj1 Just wanted to mention that in the 3.1.x and 3.0.x branches, the value was 250, so I kept the larger value of 250 rather than reducing it to 100.